tests/typescript_class/standard_private_fields.ts

```diff
@@ -0,0 +1,11 @@
+class Square {
+    #sideLength: number;
```

Comment:
maybe you can add another test case for`readonly #area: number;` as well, just to be sure that this will also be handled correctly...